### ISSUE TITLE
rnd.range(1,1) fails

### ISSUE THREAD

**Issue by Jerakin** [selimanac/defold-random#3]

**Comment by selimanac** [comment]
As expected! 
There is no number between min: 1 and max: 1, so what is the actual problem?

**Comment by Jerakin** [comment]
I would expect it to return 1. A random number between 1 and 1 is 1. The same way a random number between 1 and 2 is 1 or 2.

Most random generators I have used (mainly python) have had the following behaviour.

```
rnd.range(1, 1) -> 1 always
rnd.range(1, 0) -> ValueError

# Meaning that
if min == max:
   return min

if min > max:
  return nil

```

The change to return nil is a breaking change :)

**Comment by selimanac** [comment]
I'll implement this, but my opinion is this not a good practice. 
Trying to generate random number by using same value is pointless. 
You can simply check your min and max by using `if` as you wrote.


> if min == max:
>    return min
> 
> if min > max:
>   return nil

**Comment by Jerakin** [comment]
It's for convenience, we noticed this because in our project we use `rnd` to pick a random element from a table, the table always includes at least one.

With the previous behavior we had implemented this as `return myTable[rnd.range(1, #myTable)]`. We have have now changed that to do.
```
if #myTable == 1:
  return myTable[1]
else:
  return myTable[rnd.range(1, #myTable)]
```

Which works, even though it's a lot uglier. I am fine with either but the breaking change caught us of guard.
